content/kubeone/main/guides/cis-benchmarking/cis1.8-kubeone1.7-k8s1.27/_index.en.md

```diff
@@ -0,0 +1,569 @@
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.4: Ensure that the controller manager pod specification file ownership is set to root:root (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.5: Ensure that the scheduler pod specification file permissions are set to 600 or more restrictive (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.6: Ensure that the scheduler pod specification file ownership is set to root:root (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.7: Ensure that the etcd pod specification file permissions are set to 600 or more restrictive (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.8: Ensure that the etcd pod specification file ownership is set to root:root (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.9: Ensure that the Container Network Interface file permissions are set to 600 or more restrictive (Manual)
+
+**Result:** 🟡 Warn
+
+_The issue is under investigation to provide a fix in a future KubeOne release_
+
+---
+#### 1.1.10: Ensure that the Container Network Interface file ownership is set to root:root (Manual)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.11: Ensure that the etcd data directory permissions are set to 700 or more restrictive (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.12: Ensure that the etcd data directory ownership is set to etcd:etcd (Automated)
+
+**Result:** 🔴 Fail
+
+_The issue is under investigation to provide a fix in a future KubeOne release_
+
+---
+#### 1.1.13: Ensure that the admin.conf file permissions are set to 600 or more restrictive (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.14: Ensure that the admin.conf file ownership is set to root:root (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.15: Ensure that the scheduler.conf file permissions are set to 600 or more restrictive (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.16: Ensure that the scheduler.conf file ownership is set to root:root (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.17: Ensure that the controller-manager.conf file permissions are set to 600 or more restrictive (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.18: Ensure that the controller-manager.conf file ownership is set to root:root (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.19: Ensure that the Kubernetes PKI directory and file ownership is set to root:root (Automated)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.20: Ensure that the Kubernetes PKI certificate file permissions are set to 600 or more restrictive (Manual)
+
+**Result:** 🟢 Pass
+
+---
+#### 1.1.21: Ensure that the Kubernetes PKI key file permissions are set to 600 (Manual)
+
+**Result:** 🟢 Pass
+
+---
+### 1.2. API Server
+#### 1.2.1: Ensure that the --anonymous-auth argument is set to false (Manual)
+
+**Result:** 🔵 Not Applicable
+
+**Details:** This is mitigated by RBAC, please see https://kubernetes.io/docs/reference/access-authn-authz/authentication/#anonymous-requests
```

Comment:
I'm not sure this is sufficient. At best, we'd disable anonymous auth completely. Do we have any idea why this isn't set by KubeOne/kubeadm?